tempesta_fw/http.c

```diff
@@ -748,9 +748,23 @@ tfw_http_adjust_resp(TfwHttpResp *resp, TfwHttpReq *req)
 		return r;
 
 	r = tfw_http_add_hdr_via(hm);
-	if (r)
-		return r;
-
+	if (resp->flags & TFW_HTTP_RESP_STALE) {
+#define S_WARN_110_N "Warning"
+#define S_WARN_110_V "110 - Response is stale"
+	/* TODO: ajust for #215 */
+	TfwStr wh = {
+	.ptr = (TfwStr []) {
+		{.ptr = S_WARN_110_N, .len = SLEN(S_WARN_110_N)},
+		{.ptr = S_WARN_110_V, .len = SLEN(S_WARN_110_V)},
+}, .len = SLEN(S_WARN_110_N) + SLEN(S_WARN_110_V), .eolen = 2,
```

Comment:
This header doesn't have any variable parts. It's just a constant string. Therefore, there's no need to create a complex compound string. Also, please pay attention to the semicolon, again.